thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js

```diff
@@ -17,6 +18,14 @@ const ROOTCAUSE_SERVICE_BREAKDOWNS = "breakdowns";
 // TODO: Update module import to comply by new Ember standards
 
 export default Ember.Controller.extend({
+  session: Ember.inject.service(),
+
+  /**
+   * User ldap
+   * @type {String}
+   */
+  user: Ember.computed.reads('session.data.authenticated.name'),
```

Comment:
I think you can safely read this directly in onSessionSave()

Reply:
I believe using onSessionSave would only work for just saved report. I think she's trying to handle the case where it's already saved and we're reading the info.

Reply:
@justYves I think what @apucher means is `user` can be retrieved like `this.get('session.data.authenticated.name` inside the `onSessionSave` method, as opposed to making a CP. I'll make the change.